inc/branding/namespace.php

```diff
@@ -76,6 +77,38 @@ function add_color_scheme() {
 	);
 }
 
+/**
+ * Remove all unused default WP admin color schemes for a user.
+ *
+ * If a user has previously chosen one of the WP default admin color schemes,
+ * leave that scheme as is. Remove all other WP default admin color schemes,
+ * so that they can't be selected.
+ */
+function remove_wp_admin_color_schemes() {
+	global $_wp_admin_css_colors;
+
+	// Use color slugs as keys for a quick access.
+	$admin_color_schemes = [
+		'fresh' => true,
+		'light' => true,
+		'blue' => true,
+		'midnight' => true,
+		'sunrise' => true,
+		'ectoplasm' => true,
+		'ocean' => true,
+		'coffee' => true,
+	];
+
+	$user_admin_color = get_user_option( 'admin_color', get_current_user_id() );
+
+	// Remove all WP default admin color schemes, except the one that's selected for a user if any.
+	foreach ( $admin_color_schemes as $color_slug => $color_scheme ) {
```

Comment:
`$color_scheme` is never used so as mentioned above, no need for an associative array.